### PR TITLE
Add 243390 DT - PR 2

### PR DESCRIPTION
**Pull request by acostach**

### CODE REVIEW

**Comment by acostach** [comment]
This replaces https://github.com/balena-io-hardware/testbotsdk/pull/21

File: lib/devices.ts [hunk 1]
```diff
@@ -300,7 +301,7 @@ export class BalenaFin extends DeviceInteractor {
 			await this.powerOnFlash();
 			// etcher-sdk (power on) usboot
 			const adapters: sdk.scanner.adapters.Adapter[] = [
-				new sdk.scanner.adapters.BlockDeviceAdapter(() => false),
+				new BlockDeviceAdapter({ includeSystemDrives: () => false }),
```
Comment (rcooke-warwick) [inline lib/devices.ts:304]: i think we may have to also change this to:
```
new BlockDeviceAdapter({
					includeSystemDrives: () => false,
					unmountOnSuccess: false,
					write: true,
					direct: true,
				}),
```

**Comment by rcooke-warwick** [comment]
after that fix, this works with my fin. I still don't quite understand why it fails if we don't pipe to the file first though - but I'm happy to accept it for now if you are

**Comment by acostach** [comment]
Thanks @rcooke-warwick! I tested [fcb1e38](https://github.com/balena-io-hardware/testbotsdk/pull/36/commits/fcb1e3809aa673e532e772cd71d21a15a111cd9e) with the eMMC Pi3 and the Pi4. I will pull your commit and try again, this time to include the following 3 use-cases:
- Pi3 eMMC
- Pi4 (non-flasher)
- EBCRS A2 2G (flasher)

**Comment by rcooke-warwick** [comment]
I also just tested the Pi4, and Fin and both worked @acostach - this looks good to me and I'm happy to merge this if you are. This bump with then enable us to progress the emulated generic-arm64 tests too

**Comment by acostach** [comment]
Perfect, I'm in progress of testing the Pi3 eMMC provisioning again and the EBCRS's and will get back with my results ASAP.

**Comment by rcooke-warwick** [comment]
thanks @acostach nice work

**Review by rcooke-warwick — APPROVED** [review]

**Pull request merged**